### ISSUE TITLE
breaks on iOS with FP 7.2.1

### ISSUE THREAD

**Issue by bbbo** [flowplayer/flowplayer-background#8]
ie http://demos.flowplayer.org/plugins/background.html

Reports:
- Quickly flashes, then disappears iPhone 6s+ iOS 11.2
- starts on mine, but when you scroll to the top it stops playing iPhone 6 11.0.2

**Comment by phloxic** [comment]
I don't have iOS 11.2, only up to 11.1.
- I can repro the stop (pause) on scroll with core 7.2.1
- Does not work at all with core 7.2.0

As it works fine with 7.1.x - 7.2 seems to have introduced some kind of scroll into view behavior. I don't know how it can be turned off.

**Comment by bbbo** [comment]
yes @nnarhinen added viewport detection in 7.2 to avoid preloading of videos before they're in the viewport. Seems to conflict with some lazy load scripts as well.
